src/BlockchainTestsFiller/ValidBlocks/bcEIP3675/NoBlockRewardKeepTipFiller.yml

```diff
@@ -0,0 +1,120 @@
+# Check the effects of maxPriorityFeePerGass on the miner's (coinbase) balance
+
+
+NoBlockRewardKeepTip:
```

Comment:
I think the test you introduced below do the same and easier to read. This can be removed.